src/containers/AuthorsList.js

```diff
@@ -29,35 +29,55 @@ const _ = {
 
 class AuthorsList extends React.Component {
   static fetchData({ store }) {
-    return store.dispatch(fetchNextPageAuthors())
+    return store.dispatch(searchAuthorsIfNeeded(''))
   }
 
   constructor(props) {
     super(props)
+    this.state = {
+      whichAuthorsListToRender: 'authorsList'
+    }
   }
 
   componentDidMount() {
-    const { currentPage, setHeaderInfo, fetchNextPageAuthors } = this.props
-    if (currentPage === (REQUEST_PAGE_START_FROM -1) ) {
-      fetchNextPageAuthors()
-    }
+    const { setHeaderInfo, authorsList, searchAuthorsIfNeeded } = this.props
     setHeaderInfo({
       pageTheme: LIGHT,
       pageType: AUTHORS_LIST,
       pageTitle: PAGE_TITLE
     })
+    if ((this.state.whichAuthorsListToRender==='authorsList') && !authorsList.isFetching && authorsList.currentPage < NUMBER_OF_FIRST_RESPONSE_PAGE) {
```

Comment:
It won't fetch duplicate items since your `searchAuthorsIfNeeded ` already handle this